src/technochat/http/ochat.go

```diff
@@ -108,6 +113,8 @@ func (s *Server) ochatInit(r *http.Request) (int, interface{}, error) {
 		pplToJoin: req.MaxUsers,
 	}
 
+	log.Printf("chat inited with pplToJoin=%d", chats[chatIDStr].pplToJoin)
```

Comment:
`info: started a new chat %s for %d people`

Reply:
done